.azure-pipelines-templates/common.yml

```diff
@@ -1,37 +1,34 @@
 jobs:
-- job: BuildAndTest_${{ parameters.target }}_${{ parameters.consensus }}_${{ parameters.protocol }}_${{ parameters.suffix }}
```

Comment:
Got rid of the protocol while pushing the boulder up the hill, can't remember which hill. Anyway, it's done now.